kaiko/src/adapter.ts

```diff
@@ -46,19 +50,25 @@ const execute = (input, callback) => {
     'X-Api-Key': util.getRandomRequiredEnv('API_KEY'),
```

Comment:
This could come from `config`. Take a look to other price adapters like `amberdata` if it's of any help

Reply:
Not detected as change on the PR, but please update the way we are getting the `url`, should come from config as well.